### PR TITLE
New aspect ratios calculation

### PR DESCRIPTION
**Pull request by Archegon**
Made aspect ratio calculations adaptive to fit all display sizes.
Added auto refresh on multiple aspect ratios
Added options for Notification and Auto refresh.

New Logo and Background.

### CODE REVIEW

**Comment by TwilightMercy** [comment]
Hi,
what is the auto refresh check interval for multiple aspect ratios?

**Comment by Archegon** [comment]
> Hi, what is the auto refresh check interval for multiple aspect ratios?

Currently set to 5 seconds

**Comment by TwilightMercy** [comment]
> > Hi, what is the auto refresh check interval for multiple aspect ratios?
> 
> Currently set to 5 seconds

Can it be configurable via addon settings?

**Comment by Archegon** [comment]
> > > Hi, what is the auto refresh check interval for multiple aspect ratios?
> > 
> > 
> > Currently set to 5 seconds
> 
> Can it be configurable via addon settings?

Yes, I just commited an option to change the refresh interval, will release in a few minutes

**Comment by TwilightMercy** [comment]
Hi @Archegon, great work and thanks for your work and dedication to improve @osumoclement addon!

Can you summarize in the key points, after all the commits and changes, what are the changes/features you've added to your fork?

**Comment by osumoclement** [comment]
Hi @TwilightMercy , there's quite a bit of discussion [here](https://forum.kodi.tv/showthread.php?tid=369869&page=9), essentially there's a bunch of improvements to the frame-check algorithm. For example, it does the "take samples of aspect ratio over x period" thing to determine frame rate stability.

Been very busy so haven't had time to look at all this but will do so soon.

**Comment by osumoclement** [comment]
Hi @TwilightMercy , could you try out @Archegon 's fork and see if the improvements work well? And also that it does not introduce new problems?
